src/doc/src/guide/continuous-integration.md

```diff
@@ -49,6 +49,40 @@ breakage in nightly will not fail your overall build. Please see the
 [GitLab CI](https://docs.gitlab.com/ce/ci/yaml/README.html) for more
 information.
 
+### Github Actions
+
+To test your package on Github Actions, here is a sample `workflow.yml` file, which has to be in the directory `.github/workflows directory`:
```

Comment:
It would probably be best to use a capital `H` in the name "GitHub" throughout.

```suggestion
To test your package on GitHub Actions, here is a sample `workflow.yml` file, which has to be in the directory `.github/workflows`:
```